.gitignore

```diff
@@ -11,7 +11,7 @@ log.txt
 
 # only include appsettings for a few default environments
 appsettings.*.json
-dab-config.*.json
```

Comment:
how do we ignore anything other than a `.` after dab-config for e.g.  `dab-config-.json`?

Reply:
can we do `dab-config*.json`